Bash_scripts/HW20/roles/hw201/tasks/main.yml

```diff
@@ -0,0 +1,51 @@
+#For CentOS
+- name: Install programs on CentOS
+  import_tasks: centos.yml
+   
+#For Ubuntu
+- name: Install programs on Ubuntu
+  import_tasks: ubuntu.yml
+  
+#Install docker
+- name: Install docker
+  import_tasks: docker.yml 
+   
+#Add user Ubuntu
+- name: Add the user 'hw20' with a sh shell and '/opt/hw20' home directory Ubuntu
+  ansible.builtin.user:
```

Comment:
This is not `os_family` specific. Use variables to make user creating in one task for both OSs

Reply:
done